gcsio/src/test/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorageFileSystemIntegrationTest.java

```diff
@@ -401,15 +396,7 @@ public MethodOutcome nonExistentDeleteOutcome() {
   public void testGetAndListFileInfo() throws Exception {
     // Objects created for this test.
     String[] objectNames = {
-        "o1",
-        "o2",
-        "d0/",
-        "d1/o11",
-        "d1/o12",
-        "d1/d10/",
-        "d1/d11/o111",
-        "d2/o21",
-        "d2/o22",
+      "o1", "o2", "d0/", "d1/o11", "d1/o12", "d1/d10/", "d1/d11/o111", "d2/o21", "d2/o22",
```

Comment:
Such code snippets become less readable - we may want to disable formatting for them, or revert

Reply:
Done